tests/kafkatest/services/connect.py

```diff
@@ -229,11 +266,13 @@ def start_node(self, node):
             remote_connector_configs.append(target_file)
 
         self.logger.info("Starting Kafka Connect standalone process on " + str(node.account))
-        with node.account.monitor_log(self.LOG_FILE) as monitor:
-            cmd = self.start_cmd(node, remote_connector_configs)
-            self.logger.debug("Connect standalone command: %s", cmd)
-            node.account.ssh(cmd)
-            monitor.wait_until('Kafka Connect started', timeout_sec=60, err_msg="Never saw message indicating Kafka Connect finished startup on " + str(node.account))
+        if self.startup_mode == 'LOAD':
+            self.start_and_wait_to_load_plugins(node, 'standalone', remote_connector_configs)
+        elif self.startup_mode == 'LISTEN':
+            self.start_and_wait_to_start_listening(node, 'standalone', remote_connector_configs)
+        else:
+            # mode 'INSTANT' is implied
```

Comment:
is this really what was intended given compatibility? and do none of our tests rely on the existing behavior to wait until that message is logged? i would have expected some that use the REST API (e.g. pretty much anything distributed that submits a connector as one of its first post-start tasks) and would be flaky if we change the default to `INSTANT`

Reply:
Ok, written this way is confusing. I got confused too, before I issue the PR. Indeed, the `else` might correspond to `INSTANT` but the thing is that the member variable `startup_mode` is initialized to `LISTEN` at the top of the class. So it shouldn't be `INSTANT` in general and in any of the previous tests, unless explicitly set to that. 

But given it's confusing, I'll change it.